xbmc/dbwrappers/dataset.cpp

```diff
@@ -321,25 +335,64 @@ bool Dataset::set_field_value(const char *f_name, const field_value &value) {
   //  return false;
 }
 
+/********* GUESS SECTION START *********/
+bool Dataset::get_guess(const char *f_name) {
+  if(guess_enabled && !(ds_state == dsEdit || ds_state == dsInsert))
```

Comment:
the whole dataset.cpp is misaligned regarding spaces and tabs, pls. dont look at it for first.
I'l let an autoformatter run over this finally - but you would have every line red in this screen then